lib/src/repo.rs

```diff
@@ -902,10 +902,14 @@ impl MutableRepo {
         &mut self,
         settings: &UserSettings,
         options: RebaseOptions,
-    ) -> Result<usize, TreeMergeError> {
+    ) -> Result<RebaseCounts, TreeMergeError> {
         let result = self
             .rebase_descendants_return_rebaser(settings, options)?
-            .map_or(0, |rebaser| rebaser.into_map().len());
+            .map(|rebaser| {
+                let dropped_signatures_len = rebaser.dropped_signatures().len();
+                RebaseCounts::new(rebaser.into_map().len(), dropped_signatures_len)
```

Comment:
nit: I would inline `::new()` here as the order of `(usize, usize)` arguments is unclear.